packages/poap-sdk/src/drops/types/FetchDropsInput.ts

```diff
@@ -1,12 +1,13 @@
 import { Order, PaginationInput } from '@poap-xyz/poap-sdk';
 import { DropsSortFields } from './DropsSortFields';
 
-export interface FetchDropsInput extends PaginationInput {
-  name?: string;
+export type FetchDropsInput = PaginationInput & {
   sortField?: DropsSortFields;
   sortDir?: Order;
+  /** Drop event start date to start from the given date. */
   from?: string;
+  /** Drop event start date to end on or before the given date. */
```

Comment:
```suggestion
  /** Drop event end date to end on or before the given date. */
```


Bit repetitive the phrase.